src/core/constants.js

```diff
@@ -50,7 +50,9 @@ export const ADDON_TYPE_EXTENSION = 'extension';
 export const ADDON_TYPE_LANG = 'language';
 export const ADDON_TYPE_OPENSEARCH = 'search';
 export const ADDON_TYPE_THEME = 'persona';
+export const ADDON_TYPE_CLASSIC_THEME = 'theme';
 export const validAddonTypes = [
+  ADDON_TYPE_CLASSIC_THEME, // Remove once we don't support complete themes.
```

Comment:
One day we might start grepping `TODO` so it would probably make sense to add that to the comment so we don't forget